anndata/_io/specs/registry.py

```diff
@@ -17,6 +16,61 @@ class IOSpec(NamedTuple):
     encoding_version: str
 
 
+class NoSuchIO(KeyError):
```

Comment:
Could this live with the other errors?

Reply:
For sure!

Reply:
wait, we don’t have a place for other errors:

![grafik](https://user-images.githubusercontent.com/291575/158773452-83c37b6e-2895-4e4f-90dc-ccccdb777890.png)